lib/Ed25519VerificationKey2020.js

```diff
@@ -117,32 +159,35 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
    *   public key only).
    */
   static fromFingerprint({fingerprint} = {}) {
-    if(!fingerprint ||
-      !(typeof fingerprint === 'string' && fingerprint[0] === 'z')) {
-      throw new Error('`fingerprint` must be a multibase encoded string.');
-    }
-
-    // skip leading `z` that indicates base58 encoding
-    const buffer = base58btc.decode(fingerprint.substr(1));
-
-    // buffer is: 0xed 0x01 <public key bytes>
-    if(buffer[0] === 0xed && buffer[1] === 0x01) {
-      return new Ed25519VerificationKey2020({
-        publicKeyMultibase: `z${base58btc.encode(buffer.slice(2))}`
-      });
-    }
-
-    throw new Error(`Unsupported fingerprint "${fingerprint}".`);
+    return new Ed25519VerificationKey2020({publicKeyMultibase: fingerprint});
   }
 
   get _publicKeyBuffer() {
-    return this.publicKeyMultibase &&
+    if(!this.publicKeyMultibase) {
+      return false;
```

Comment:
This previously would have returned `undefined` instead. It's weird to return `false` anyway.

```suggestion
      return;
```